core/client/fs/src/test/java/alluxio/client/file/FileSystemTestUtils.java

```diff
@@ -129,6 +132,20 @@ public static void createByteFile(FileSystem fs, String fileName, WriteType writ
     }
   }
 
+  /**
+   * Loads a file into Alluxio by reading it.
+   *
+   * @param fs a {@link FileSystem}
+   * @param fileName the name of the file to load
+   */
+  public static void loadFile(FileSystem fs, String fileName) {
+    try (FileInStream is = fs.openFile(new AlluxioURI(fileName))) {
```

Comment:
Should this explicitly use a `CACHE` read type?

Reply:
done